pkg/trace/config/config_test.go

```diff
@@ -41,7 +41,9 @@ func TestConfigHostname(t *testing.T) {
 			fallbackHostnameFunc = os.Hostname
 		}()
 		_, err := Load("./testdata/site_override.yaml")
-		assert.Contains(err.Error(), "nor from OS")
+		if err != nil {
```

Comment:
My mistake, when I ran these tests early on, it would throw a panic error from calling `err.Error()` but that might've derived from my changes. I ran it again with the original version (without the nil check) and it works, reverted this change.